lib/src/components/NxCombobox/NxCombobox.tsx

```diff
@@ -99,6 +99,9 @@ function NxComboboxRender<T extends string | number | DataItem<string | number,
         doSearch(value);
       }
     }
+    if (matches.length || value) {
```

Comment:
Is this condition necessary?

Reply:
> Is this condition necessary?

Thanks for pointing that out- no, it was for an old iteration and should have been removed. Removed here: [b9404ed](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/b9404edb82109eba303d62525fad164dc551dd4e)